pytest.ini

```diff
@@ -1,2 +1,3 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = django_test_settings
+addopts = --random-order
```

Comment:
Pytest config shouldn't be defined in multiple places. I would remove the `pytest.ini` file and just define it in the `setup.cfg` file.